app/assets/javascripts/app/utils/auto-logout.js

```diff
@@ -1,5 +1,3 @@
 export default () => {
-  window.onbeforeunload = null;
```

Comment:
@zachmargolis Does the `onbeforeunload = null` target a specific browser? In chrome, at least, I don't see any browser alerts before the page is reloaded.

I tested the following:

- reloading the page manually
- filling in some form fields and reloading manually
- doing the same as above but reloading via the 'keep me signed in' option in the modal.